src/go/configgenerator/listener_generator_test.go

```diff
@@ -1808,10 +1812,12 @@ func TestMakeHttpConMgr(t *testing.T) {
 				`,
 		},
 		{
-			desc: "Generate HttpConMgr when tracing is disabled",
+			desc: "Generate HttpConMgr when tracing is enabled",
 			opts: options.ConfigGeneratorOptions{
 				CommonOptions: options.CommonOptions{
-					DisableTracing: true,
+					DisableTracing:      false,
+					TracingProjectId:    "test-project",
+					TracingSamplingRate: 1,
```

Comment:
So only the sample rate doesn't work in bootstrap?  
So the current sample rate in our code is 1?

Reply:
Yup, apparently setting tracing in bootstrap was deprecated sometime within the last year. Envoy defaults to sample rate of 1 then.

Reply:
Does that mean all the ESPv2 users are using 100% tracing and no one find/complain till now?

Reply:
Yup, for at least 5 months from when the deprecation happened.

Reply:
Since tracing all the requests has been the (unintended) default behavior for many months now (possibly since project start?), it may make sense for the fix to set 1.0 as the new default, in case your other users have been relying on it, and to maintain backwards compatibility. Or you could just highlight it as a breaking change by making 0.001 work as intended, up to you.

Reply:
Good question, but I think it should be fine:
1) The current behavior does not match the [documented behavior](https://cloud.google.com/endpoints/docs/openapi/specify-esp-v2-startup-options#tracing). In this case, making the bug fix with a breaking change is OK.
2) The breaking change does not affect client-facing behavior. If anything, requests will be slightly faster. So the impact of the breaking change is minimal.
3) The majority of Cloud Endpoints users are ESPv1 on GKE, which ESPv2 will "very soon" support. Our aim is to make the migration as easy as possible. So we don't want to change any flag defaults values that ESPv1 uses.